astrowidgets/tests/test_api.py

```diff
@@ -100,8 +100,15 @@ def test_reset_markers():
     # (it also doesn't *do* anything...)
     image.reset_markers()
     assert image.get_markers() is None
-
-    # TODO add test of actually removing markers...
+    table = Table(data=np.random.randint(0, 100, [5, 2]),
+                  names=['x', 'y'], dtype=('int', 'int'))
+    image.add_markers(table, x_colname='x', y_colname='y',
+                      skycoord_colname='coord', marker_name='test')
+    image.add_markers(table, x_colname='x', y_colname='y',
+                      skycoord_colname='coord', marker_name='test2')
+    image.reset_markers()
+    assert image._viewer.canvas.tags.get('test') is None
```

Comment:
Could you please change these to check the results of calls to `image.get_markers` instead of calling the underlying ginga viewer? I'd like to try to make sure that the tests rely as much as possible on the public API rather than the internals of the particular viewer.

Reply:
Hi! sure! I have updated the code, thanks!